kafka-mixin/alerts/KafkaAlerts.yml

```diff
@@ -35,7 +35,7 @@ groups:
         severity: critical
       annotations:
         description: 'Kafka {{ $labels.instance }}: {{ $value }} unclean leader elections'
-        summary: '{{ $value }} unclean partition leader elections in the cluster reported in the last 1 minute interval. When unclean leader election is held among out-of-sync replicas, there is a possibility of data loss if any messages were not synced prior to the loss of the former leader. So if the number of unclean elections is greater than 0, investigate broker logs to determine why leaders were re-elected, and look for WARN or ERROR messages. Consider setting the broker configuration parameter unclean.leader.election.enable to false so that a replica outside of the set of in-sync replicas is never elected leader.'
```

Comment:
Perhaps, using templates in summary should be avoided where possible so alert can be 
" it can be used as a common “headline” for all alerts in the notification template"
https://monitoring.mixins.dev/#guidelines-for-alert-names-labels-and-annotations